code/modules/paperwork/clipboard.dm

```diff
@@ -33,156 +35,182 @@
+	var/obj/item/I = user.get_active_hand()
+	if(I != src && IS_PEN(I))
+		return I
+	for(I in user.get_held_items()) //Its pretty likely the current held thing is the clipdboard
+		if(IS_PEN(I))
+			return I //In that case pick the first pen item we're holding
+
+/obj/item/clipboard/proc/add_pen(var/obj/item/I, var/mob/user)
+	if(!stored_pen && I.w_class <= ITEM_SIZE_TINY && IS_PEN(I) && user.unEquip(I, src))
+		stored_pen = I
+		to_chat(user, SPAN_NOTICE("You slot \the [I] into \the [src]."))
+		return TRUE
+	else if(stored_pen)
+		to_chat(user, SPAN_WARNING("There is already \a [stored_pen] in \the [src]."))
+	else if(I.w_class > ITEM_SIZE_TINY)
+		to_chat(user, SPAN_WARNING("\The [I] is too big to fit in \the [src]."))
+
+/obj/item/clipboard/proc/remove_pen(var/mob/user)
+	if(stored_pen && user.put_in_hands(stored_pen))
+		to_chat(user, SPAN_NOTICE("You pull your trusty [stored_pen] from your [src]."))
+		. = stored_pen
+		stored_pen = null
+		return .
+	else if(!stored_pen)
+		to_chat(user, SPAN_WARNING("There is no pen in \the [src]."))
+	else
+		to_chat(user, SPAN_WARNING("Your hands are full.")) 
+
+/obj/item/clipboard/OnTopic(mob/user, href_list, datum/topic_state/state)
+	. = ..()
+	var/obj/item/tpaper = top_paper()
+
+	if(href_list["pen"] && remove_pen(user))
+		. = TOPIC_HANDLED | TOPIC_REFRESH
+
+	else if(href_list["addpen"] && add_pen(get_user_pen(user), user))
+		. = TOPIC_HANDLED | TOPIC_REFRESH
+
+	else if(href_list["write"])
+		if(tpaper)
+			var/obj/item/I = get_user_pen(user)
+			//We can also use the stored pen if we have one and a free hand
+			if(!I && IS_PEN(stored_pen))
+				I = remove_pen(user)
+			else if(!I)
+				to_chat(user, SPAN_WARNING("You don't have a pen!"))
+
+			if(I)
+				tpaper.attackby(I, user)
+				. = TOPIC_HANDLED | TOPIC_REFRESH
+		else
+			. = TOPIC_NOACTION
+
+	else if(href_list["remove"])
+		var/obj/item/P = locate(href_list["remove"])
+		if(P && user.put_in_hands(P))
+			papers.Remove(P)
+			. = TOPIC_HANDLED | TOPIC_REFRESH
+
+	else if(href_list["rename"])
+		var/obj/item/O = locate(href_list["rename"])
+		if(istype(O, /obj/item/paper))
+			var/obj/item/paper/to_rename = O
+			to_rename.rename()
+			. = TOPIC_HANDLED | TOPIC_REFRESH
+
+		else if(istype(O, /obj/item/photo))
+			var/obj/item/photo/to_rename = O
+			to_rename.rename()
+			. = TOPIC_HANDLED | TOPIC_REFRESH
+
+		else
+			. = TOPIC_NOACTION
+
+	else if(href_list["examine"])
+		var/obj/item/P = locate(href_list["examine"])
+
+		if(istype(P, /obj/item/paper))
+			var/obj/item/paper/PP = P
+			PP.show_content(user)
+			. = TOPIC_HANDLED
+
+		else if(istype(P, /obj/item/photo))
+			var/obj/item/photo/PP = P
+			PP.show(user)
+			. = TOPIC_HANDLED
+
+	//Update everything
+	if(. & TOPIC_REFRESH)
+		attack_self(user)
 		update_icon()
-	return
 
 /obj/item/clipboard/ebony
 	material = /decl/material/solid/wood/ebony
 
 /obj/item/clipboard/steel
 	material = /decl/material/solid/metal/steel
-	material = /decl/material/solid/metal/steel
```

Comment:
Wonder how this happened? Merge artifact/find and replace mistake?

Reply:
Seems like a merge artifact to me. Sometimes during rebase it just can't seem to be able to tell when two competing changes are actually identical. Or maybe its just a human error.